src/core/moves/MetropolisHastingsMove.cpp

```diff
@@ -320,11 +424,34 @@ void MetropolisHastingsMove::performMcmcMove( double prHeat, double lHeat, doubl
         {
             if ( the_node->isClamped() )
             {
-                ln_likelihood_ratio += the_node->getLnProbabilityRatio();
+                try
+                {
+                    ln_likelihood_ratio += the_node->getLnProbabilityRatio();
+                }
+                catch (RbException e)
```

Comment:
catching exception by value.

Reply:
By value is like `catch (RbException e)`.
By reference is like `catch (RbException& e)`.
Sorry, I was referring to a comment above, but it was still kind of unclear.

Reply:
Changed with the previous commit.